### ISSUE TITLE
Need to update settingUp.md 

### ISSUE THREAD

**Issue by aravindputrevu** [TEAMMATES/teammates#6272]
In the settingUp.md, It is recommended to use Eclipse Luna for setting up dev environment for TEAMMATES. However, I used the latest Eclipse IDE Neon to set up dev environment and it worked.

So need to change documentation accordingly.

**Comment by wkurniawan07** [comment]
I will update this accordingly in #6257. Thanks for letting us know that Eclipse Neon works as well :+1: I will add your name to our contributors' list prior to our next public release.
May I also confirm that the only difference is that the Google Plugin to be used is version `4.6`?

**Comment by aravindputrevu** [comment]
Yes.
I used :
```
Eclipse IDE 
Version: Neon.2 Release (4.6.2)
Build id: 20161208-0600
Google Plugin 4.6.
```

Thank you @wkurniawan07

**Comment by wkurniawan07** [comment]
Fixed in #6257.
